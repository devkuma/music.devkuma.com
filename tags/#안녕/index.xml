<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Music – /#안녕</title>
    <link>https://music.devkuma.com/tags/#%EC%95%88%EB%85%95/</link>
    <image>
      <url>https://music.devkuma.com/tags/#%EC%95%88%EB%85%95/logo/180x180.jpg</url>
      <title>/#안녕</title>
      <link>https://music.devkuma.com/tags/#%EC%95%88%EB%85%95/</link>
    </image>
    <description>Recent content in /#안녕 on Music</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    
	  <atom:link href="https://music.devkuma.com/tags/#%EC%95%88%EB%85%95/index.xml" rel="self" type="application/rss+xml" />
    
    
      
        
      
    
    
    <item>
      <title></title>
      <link>https://music.devkuma.com/track/0023/509225b6/1a0dc7f7/</link>
      <pubDate>Sun, 29 Oct 2023 12:17:00 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/0023/509225b6/1a0dc7f7/</guid>
      <description>
        
        
        &lt;p&gt;작사: 이푸름, 유상균, 리트너, 강재수&lt;br&gt;
/ KY: &lt;a href=&#34;https://www.youtube.com/@TJKaraoke/search?query=82498&#34; target=&#34;_black&#34;&gt;82498&lt;/a&gt;&lt;/p&gt;
&lt;p&gt;너에게 다른 사람이&lt;br&gt;
생겼다는 소식을 듣게 되었어&lt;br&gt;
난 하루 종일 복잡한 마음이라서&lt;br&gt;
더 아무것도 할 수 없었어&lt;/p&gt;
&lt;p&gt;날 보며 웃는 너의 행복했던 모습이&lt;br&gt;
이제는 아픔이 되고&lt;br&gt;
그렇게 니 생각은 자꾸만 깊어져&lt;br&gt;
추억 속 어딘가로 날 데려가는데&lt;/p&gt;
&lt;p&gt;이젠 너에게&lt;br&gt;
사랑을 말할 수 없고&lt;br&gt;
이별도 쉽게 못하고&lt;/p&gt;
&lt;p&gt;믿기지가 않아&lt;br&gt;
난 어떡해야 해&lt;/p&gt;
&lt;p&gt;우리의 사랑이 끝났다 해도&lt;br&gt;
다신 볼 수 없어도&lt;br&gt;
하나만 기억해 줘&lt;br&gt;
함께 한 시절&lt;/p&gt;
&lt;p&gt;우리가 함께했던 모든 순간순간이&lt;br&gt;
이제는 돌아오지 않는데&lt;br&gt;
덩그러니 남겨진 너의 사진 보면&lt;br&gt;
추억 속 그곳에선 함께인 것만 같아&lt;/p&gt;
&lt;p&gt;이젠 너에게&lt;br&gt;
사랑을 말할 수 없고&lt;br&gt;
이별도 쉽게 못하고&lt;/p&gt;
&lt;p&gt;믿기지가 않아&lt;br&gt;
난 어떡해야 해&lt;/p&gt;
&lt;p&gt;우리의 사랑이 끝났다 해도&lt;br&gt;
다신 볼 수 없어도&lt;br&gt;
하나만 기억해 줘&lt;br&gt;
함께 한 시절&lt;/p&gt;
&lt;p&gt;다 끝인가 봐&lt;br&gt;
되돌릴 수 없는 걸&lt;br&gt;
난 알고 있지만&lt;/p&gt;
&lt;p&gt;너에게 사랑을 말할 수 없다&lt;/p&gt;
&lt;p&gt;내 맘은 아직 널 그리는데&lt;br&gt;
여전히 난 너에게&lt;br&gt;
스쳐버린 사람 그런 추억일까&lt;/p&gt;
&lt;p&gt;시간이 저 멀리 널 데려가도&lt;br&gt;
더는 볼 수 없어도&lt;br&gt;
하나만 기억해줘&lt;br&gt;
함께 한 시절&lt;/p&gt;

      </description>
      
      <category>/#안녕</category>
      
    </item>
    
    <item>
      <title></title>
      <link>https://music.devkuma.com/track/0023/509225b6/fafefeb5/</link>
      <pubDate>Sun, 29 Oct 2023 12:16:57 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/0023/509225b6/fafefeb5/</guid>
      <description>
        
        
        &lt;p&gt;작사: JUNE, CHRIS지훈, 유상균, 리트너&lt;br&gt;
/ KY: &lt;a href=&#34;https://www.youtube.com/@TJKaraoke/search?query=82894&#34; target=&#34;_black&#34;&gt;82894&lt;/a&gt;&lt;/p&gt;

&lt;div style=&#34;position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;&#34;&gt;
  &lt;iframe src=&#34;https://www.youtube.com/embed/SS5mdirbJKQ&#34; style=&#34;position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;&#34; allowfullscreen title=&#34;YouTube Video&#34;&gt;&lt;/iframe&gt;
&lt;/div&gt;

&lt;hr&gt;
&lt;p&gt;꿈에 너의 번홀&lt;br&gt;
연락처에 다시 올렸어&lt;br&gt;
이미 바뀐지 오랜데&lt;/p&gt;
&lt;p&gt;너의 번홀 등록하니&lt;br&gt;
떠오른 너의 얼굴&lt;br&gt;
참 오랜만에 보았어&lt;/p&gt;
&lt;p&gt;모두 잠든 밤&lt;br&gt;
너에게 무슨 말을 건넬까&lt;br&gt;
어차피 꿈인데&lt;br&gt;
왜 넌 망설이니&lt;/p&gt;
&lt;p&gt;꿈에서 본 너에게&lt;br&gt;
말하다 꿈을 깰 까봐&lt;br&gt;
아무 말 못한 채 바라보고 있어&lt;/p&gt;
&lt;p&gt;몇 번의 이별을&lt;br&gt;
해야 괜찮니&lt;br&gt;
이번엔 안되니&lt;br&gt;
그만 아프고 싶은데&lt;/p&gt;
&lt;p&gt;눈물이 흐른다&lt;br&gt;
내 기억 추억 모두 흐른다 아~&lt;br&gt;
웃는다 이별이다&lt;br&gt;
안녕&lt;/p&gt;
&lt;p&gt;드디어 너에게&lt;br&gt;
도착한 설레는 메시지&lt;br&gt;
안녕 잘 지냈니&lt;br&gt;
왠지 아픈 그 말&lt;/p&gt;
&lt;p&gt;너에게 잘해주지 못한&lt;br&gt;
일만 떠올리다&lt;br&gt;
나도 모를 전활 네게 걸고 있어&lt;/p&gt;
&lt;p&gt;몇 번의 이별을&lt;br&gt;
해야 괜찮니&lt;br&gt;
이번엔 안되니&lt;br&gt;
그만 아프고 싶은데&lt;/p&gt;
&lt;p&gt;눈물이 흐른다&lt;br&gt;
내 기억 추억 모두 흐른다 아아-&lt;br&gt;
이제 그만 정말 그만&lt;/p&gt;
&lt;p&gt;안녕&lt;br&gt;
이렇게 널 여러 번 보냈어&lt;br&gt;
이제 그만 널 잊고 싶은데&lt;/p&gt;
&lt;p&gt;꿈에서도&lt;br&gt;
우린 이별하지 못하고&lt;br&gt;
이렇게 가슴 터지게 눈물만 흐른다&lt;/p&gt;
&lt;p&gt;몇 번의 이별을&lt;br&gt;
해도 난 너야&lt;br&gt;
다시 제자리야&lt;br&gt;
그저 내가 사는 이유&lt;/p&gt;
&lt;p&gt;너야&lt;br&gt;
아무리 이별&lt;br&gt;
요란하게 해봐도&lt;br&gt;
또 그 자리&lt;br&gt;
절대 비워지지 않는 네 자리&lt;/p&gt;
&lt;p&gt;(End)&lt;/p&gt;

      </description>
      
      <category>/#안녕</category>
      
    </item>
    
    <item>
      <title>먼데이키즈,#안녕 - 이 세상 끝까지</title>
      <link>https://music.devkuma.com/track/1106/ea7358e0/1e2dccab/</link>
      <pubDate>Sun, 22 Oct 2023 02:28:07 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/1106/ea7358e0/1e2dccab/</guid>
      <description>
        
        
        &lt;p&gt;작곡: 빅가이로빈 / 작사: 빅가이로빈&lt;br&gt;
KY: &lt;a href=&#34;https://www.youtube.com/@KARAOKEKY/search?query=93004&#34; target=&#34;_black&#34;&gt;93004&lt;/a&gt; / KY: &lt;a href=&#34;https://www.youtube.com/@TJKaraoke/search?query=83739&#34; target=&#34;_black&#34;&gt;83739&lt;/a&gt;&lt;/p&gt;
&lt;p&gt;어떤 말을 해 볼까&lt;br&gt;
생각보다 떨려서&lt;br&gt;
머뭇거리는 내가&lt;br&gt;
참 어색해&lt;br&gt;
서로를 잘 몰랐던&lt;br&gt;
우리가 여기&lt;br&gt;
두 손을 마주 잡고서&lt;br&gt;
평생을 약속하네요&lt;br&gt;
어느새 시간이 흘러서&lt;br&gt;
지금을 추억하는 날에&lt;br&gt;
우리는 어떤 미소들로&lt;br&gt;
바라보게 될까&lt;br&gt;
사랑하는&lt;br&gt;
그대여&lt;br&gt;
이젠 매일&lt;br&gt;
내 옆에 있어요&lt;br&gt;
분명하게 기억해&lt;br&gt;
우리의 처음을&lt;br&gt;
그대만을 따라서&lt;br&gt;
내가 할 수 있는&lt;br&gt;
작은 무언가 있기를&lt;br&gt;
영원할 우릴 위해&lt;br&gt;
이 세상&lt;br&gt;
끝까지&lt;br&gt;
거센 바람 휘몰아쳐도&lt;br&gt;
고단함에 얼굴 붉혀도&lt;br&gt;
그대 내 손 놓치지 말고&lt;br&gt;
함께해 줄래요&lt;br&gt;
사랑하는&lt;br&gt;
그대여&lt;br&gt;
이젠 매일&lt;br&gt;
내 옆에 있어요&lt;br&gt;
분명하게 기억해&lt;br&gt;
우리의 처음을&lt;br&gt;
그대만을 따라서&lt;br&gt;
내가 할 수 있는&lt;br&gt;
작은 무언가 있기를&lt;br&gt;
영원할 우릴 위해&lt;br&gt;
이 세상&lt;br&gt;
끝까지&lt;br&gt;
험난한 세상 속에&lt;br&gt;
내가 버틸 수 있던 건&lt;br&gt;
나를 바라봐 주는&lt;br&gt;
그대 때문에&lt;br&gt;
혼자여서 외롭던&lt;br&gt;
그랬던 난 이제 없어&lt;br&gt;
내 마음에&lt;br&gt;
그대가 가득 차 있으니&lt;br&gt;
그대&lt;br&gt;
내게 와 줘서&lt;br&gt;
더는 아무것도&lt;br&gt;
두렵지 않게 된걸요&lt;br&gt;
이제는 내가 그댈&lt;br&gt;
채워 주려 해요&lt;/p&gt;

      </description>
      
      <category>먼데이키즈</category>
      
      <category>/#안녕</category>
      
    </item>
    
    <item>
      <title>먼데이 키즈(Monday Kiz),#안녕 - 이 세상 끝까지</title>
      <link>https://music.devkuma.com/track/1106/17d585b9/1e2dccab/</link>
      <pubDate>Sun, 15 Oct 2023 23:58:05 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/1106/17d585b9/1e2dccab/</guid>
      <description>
        
        
        
&lt;div style=&#34;position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;&#34;&gt;
  &lt;iframe src=&#34;https://www.youtube.com/embed/qrBMbvNII9s&#34; style=&#34;position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;&#34; allowfullscreen title=&#34;YouTube Video&#34;&gt;&lt;/iframe&gt;
&lt;/div&gt;

&lt;hr&gt;
&lt;p&gt;어떤 말을 해볼까&lt;br&gt;
생각보다 떨려서&lt;br&gt;
머뭇거리는 내가 참 어색해&lt;/p&gt;
&lt;p&gt;서로를 잘 몰랐던&lt;br&gt;
우리가 여기&lt;br&gt;
두 손을 마주 잡고서&lt;br&gt;
평생을 약속하네요&lt;/p&gt;
&lt;p&gt;어느새 시간이 흘러서&lt;br&gt;
지금을 추억하는 날에&lt;br&gt;
우리는 어떤 미소들로&lt;br&gt;
바라보게 될까&lt;/p&gt;
&lt;p&gt;사랑하는 그대여&lt;br&gt;
이젠 매일 내 옆에 있어요&lt;br&gt;
분명하게 기억해&lt;br&gt;
우리의 처음을&lt;/p&gt;
&lt;p&gt;그대만을 따라서&lt;br&gt;
내가 할 수 있는&lt;br&gt;
작은 무언가 있기를&lt;br&gt;
영원할 우릴 위해&lt;br&gt;
이 세상 끝까지&lt;/p&gt;
&lt;p&gt;거센 바람 휘몰아쳐도&lt;br&gt;
고단함에 얼굴 붉혀도&lt;br&gt;
그대 내 손 놓치지 말고&lt;br&gt;
함께해 줄래요&lt;/p&gt;
&lt;p&gt;사랑하는 그대여&lt;br&gt;
이젠 매일 내 옆에 있어요&lt;br&gt;
분명하게 기억해&lt;br&gt;
우리의 처음을&lt;/p&gt;
&lt;p&gt;그대만을 따라서&lt;br&gt;
내가 할 수 있는&lt;br&gt;
작은 무언가 있기를&lt;br&gt;
영원할 우릴 위해&lt;br&gt;
이 세상 끝까지&lt;/p&gt;
&lt;p&gt;험난한 세상 속에&lt;br&gt;
내가 버틸 수 있던 건&lt;br&gt;
나를 바라봐 주는&lt;br&gt;
그대 때문에&lt;/p&gt;
&lt;p&gt;혼자여서 외롭던&lt;br&gt;
그랬던 난 이제 없어&lt;br&gt;
내 마음에 그대가&lt;br&gt;
가득 차 있으니&lt;/p&gt;
&lt;p&gt;그대 내게 와줘서&lt;br&gt;
더는 아무것도&lt;br&gt;
두렵지 않게 된 걸요&lt;br&gt;
이제는 내가 그댈&lt;br&gt;
채워주려 해요&lt;/p&gt;

      </description>
      
      <category>먼데이 키즈(Monday Kiz)</category>
      
      <category>/#안녕</category>
      
    </item>
    
  </channel>
</rss>
