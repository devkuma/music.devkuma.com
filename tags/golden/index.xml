<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Music – Golden</title>
    <link>https://music.devkuma.com/tags/golden/</link>
    <image>
      <url>https://music.devkuma.com/tags/golden/logo/180x180.jpg</url>
      <title>Golden</title>
      <link>https://music.devkuma.com/tags/golden/</link>
    </image>
    <description>Recent content in Golden on Music</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    
	  <atom:link href="https://music.devkuma.com/tags/golden/index.xml" rel="self" type="application/rss+xml" />
    
    
      
        
      
    
    
    <item>
      <title>Golden - Hate Everything (Korean Ver.)</title>
      <link>https://music.devkuma.com/track/0047/34538311/e0c0aeb6/</link>
      <pubDate>Mon, 16 Oct 2023 00:43:57 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/0047/34538311/e0c0aeb6/</guid>
      <description>
        
        
        &lt;p&gt;작곡: .., 모키오, Golden / 작사: Golden&lt;br&gt;
KY: &lt;a href=&#34;https://www.youtube.com/@KARAOKEKY/search?query=22011&#34; target=&#34;_black&#34;&gt;22011&lt;/a&gt;&lt;/p&gt;
&lt;p&gt;그냥 행복해 보이는데&lt;br&gt;
내 머릿속의 사진 속엔&lt;br&gt;
가장 예쁜 dress 입고&lt;br&gt;
웃고 있는 니 곁의 나&lt;br&gt;
비가&lt;br&gt;
슬프게 내릴 때면&lt;br&gt;
니가 싫어했던&lt;br&gt;
그 영화처럼&lt;br&gt;
회상만 가득한 날&lt;br&gt;
모든 게 싫은 그런 날&lt;br&gt;
이 노래를 부르는 것도&lt;br&gt;
강한 척하는 것도&lt;br&gt;
너 떠난 것도&lt;br&gt;
내 단점들도&lt;br&gt;
너 새로 만난 사랑도&lt;br&gt;
hate everything&lt;br&gt;
Just hate everything&lt;br&gt;
right now&lt;br&gt;
아무리 줘도 부족한걸&lt;br&gt;
알면서도 난 그렇게 널&lt;br&gt;
영원히 옆에 두려고&lt;br&gt;
아파하는 날 버려두고&lt;br&gt;
비가&lt;br&gt;
슬프게 내릴 때면&lt;br&gt;
니가 싫어했던&lt;br&gt;
그 영화처럼&lt;br&gt;
후회만 가득한 날&lt;br&gt;
모든 게 싫은 그런 날&lt;br&gt;
이 노래를 부르는 것도&lt;br&gt;
강한 척하는 것도&lt;br&gt;
너 떠난 것도&lt;br&gt;
내 단점들도&lt;br&gt;
니 새로 만난 사랑도&lt;br&gt;
hate everything&lt;br&gt;
Just hate everything&lt;br&gt;
right now&lt;br&gt;
되돌릴 수 있다면&lt;br&gt;
다 지워 버릴 수 있다면&lt;br&gt;
아름다웠던&lt;br&gt;
시간들&lt;br&gt;
Now it&amp;rsquo;s just&lt;br&gt;
killing me oh baby&lt;br&gt;
이 노래를 부르는 것도&lt;br&gt;
강한 척하는 것도&lt;br&gt;
너 떠난 것도&lt;br&gt;
내 단점들도&lt;br&gt;
너 새로 만난 사랑도&lt;br&gt;
hate everything&lt;br&gt;
Just hate everything&lt;br&gt;
right now&lt;br&gt;
Hate&lt;br&gt;
everything&lt;br&gt;
Just hate everything&lt;br&gt;
right now&lt;/p&gt;

      </description>
      
      <category>Golden</category>
      
    </item>
    
    <item>
      <title>박재범,Golden,pH-1 - Afternoon</title>
      <link>https://music.devkuma.com/track/1107/b947c4b7/5b4d76fe/</link>
      <pubDate>Sun, 15 Oct 2023 23:57:52 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/1107/b947c4b7/5b4d76fe/</guid>
      <description>
        
        
        &lt;p&gt;작곡: Bronze, 박재범, Go.. / 작사: 박재범, Golden, pH-1&lt;br&gt;
KY: &lt;a href=&#34;https://www.youtube.com/@KARAOKEKY/search?query=22189&#34; target=&#34;_black&#34;&gt;22189&lt;/a&gt;&lt;/p&gt;
&lt;p&gt;Baby 눈을 감아&lt;br&gt;
이제 더는 없어 드라마&lt;br&gt;
어딜 가도 너만 찾아&lt;br&gt;
손을 잡고 멀리 떠나자&lt;br&gt;
I&amp;rsquo;ma love you&lt;br&gt;
in the morning&lt;br&gt;
And&lt;br&gt;
in the evening&lt;br&gt;
And even&lt;br&gt;
in the afternoon&lt;br&gt;
In the morning&lt;br&gt;
and in the evening&lt;br&gt;
And even&lt;br&gt;
in the afternoon&lt;br&gt;
언제부턴지&lt;br&gt;
참지 못하고&lt;br&gt;
이렇게 나 천천히&lt;br&gt;
빠져 가는 건지&lt;br&gt;
침착하려고&lt;br&gt;
잘 보이려고&lt;br&gt;
노력하는 내 모습&lt;br&gt;
어떨지 몰라도&lt;br&gt;
설명할 수 없는&lt;br&gt;
감정들뿐인데 난&lt;br&gt;
If you ever leave&lt;br&gt;
모든 걸 잃은 듯&lt;br&gt;
슬퍼할 내 모습이&lt;br&gt;
oh no&lt;br&gt;
Baby 눈을 감아&lt;br&gt;
이제 더는 없어 드라마&lt;br&gt;
어딜 가도 너만 찾아&lt;br&gt;
손을 잡고 멀리 떠나자&lt;br&gt;
I&amp;rsquo;ma love you&lt;br&gt;
in the morning&lt;br&gt;
And&lt;br&gt;
in the evening&lt;br&gt;
And even&lt;br&gt;
in the afternoon&lt;br&gt;
In the morning&lt;br&gt;
and in the evening&lt;br&gt;
And even&lt;br&gt;
in the afternoon&lt;br&gt;
Issa vibe&lt;br&gt;
분위기 잡아 봐 girl&lt;br&gt;
It&amp;rsquo;s alright&lt;br&gt;
고민을 내려놔 oh&lt;br&gt;
나는 바라는 게 없어&lt;br&gt;
니 마음뿐&lt;br&gt;
Yeah my love is real&lt;br&gt;
간절히 기도해&lt;br&gt;
Can you love me too?&lt;br&gt;
난 니가 필요해&lt;br&gt;
널 떠날 일은 없어&lt;br&gt;
my girl 걱정 마&lt;br&gt;
설명할 수 없는&lt;br&gt;
감정들뿐인데 난&lt;br&gt;
If you ever leave&lt;br&gt;
모든 걸 잃은 듯&lt;br&gt;
슬퍼할 내 모습이&lt;br&gt;
oh no&lt;br&gt;
I&amp;rsquo;ma love you&lt;br&gt;
like I&amp;rsquo;m Marley&lt;br&gt;
지금 아주 멀리&lt;br&gt;
떠나자&lt;br&gt;
둘만을 위해서&lt;br&gt;
준비된 party&lt;br&gt;
딴생각은 트렁크 안에&lt;br&gt;
drop top&lt;br&gt;
Got us&lt;br&gt;
whippin&amp;rsquo;&lt;br&gt;
Thru the city&lt;br&gt;
like a cop car&lt;br&gt;
Got plenty of cash&lt;br&gt;
in the bag&lt;br&gt;
하고픈 거&lt;br&gt;
다 해&lt;br&gt;
내 옆에 있을 때&lt;br&gt;
모두 현실이 돼&lt;br&gt;
Welcome&lt;br&gt;
to the lifestyle&lt;br&gt;
You been dreaming&lt;br&gt;
like forever&lt;br&gt;
사진 대신&lt;br&gt;
두 눈 속에 새겨&lt;br&gt;
Baby 눈을 감아&lt;br&gt;
이제 더는 없어 드라마&lt;br&gt;
어딜 가도 너만 찾아&lt;br&gt;
손을 잡고 멀리 떠나자&lt;br&gt;
I&amp;rsquo;ma love you&lt;br&gt;
in the morning&lt;br&gt;
And&lt;br&gt;
in the evening&lt;br&gt;
And even&lt;br&gt;
in the afternoon&lt;br&gt;
In the morning&lt;br&gt;
and in the evening&lt;br&gt;
And even&lt;br&gt;
in the afternoon&lt;/p&gt;

      </description>
      
      <category>박재범</category>
      
      <category>Golden</category>
      
      <category>pH-1</category>
      
    </item>
    
    <item>
      <title>pH-1,Golden,서동현(BIG Naughty),박재범 - Oscar</title>
      <link>https://music.devkuma.com/track/0070/4506f92d/b2bf6869/</link>
      <pubDate>Sun, 15 Oct 2023 21:15:12 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/0070/4506f92d/b2bf6869/</guid>
      <description>
        
        
        &lt;p&gt;작곡: Golde.., GXXD, pH-1 / 작사: BIG.., pH-1, Golden&lt;br&gt;
KY: &lt;a href=&#34;https://www.youtube.com/@KARAOKEKY/search?query=28158&#34; target=&#34;_black&#34;&gt;28158&lt;/a&gt;&lt;/p&gt;
&lt;p&gt;Every good day start&lt;br&gt;
with a good yawn&lt;br&gt;
잠을 푹 잤더니&lt;br&gt;
시간은 두 시야&lt;br&gt;
밖에 나갈 준비 하지&lt;br&gt;
너를 만나러&lt;br&gt;
너와 sipping 하는&lt;br&gt;
커피는 너무 달아&lt;br&gt;
넌 당부했지&lt;br&gt;
오해하지 마&lt;br&gt;
사귈 마음 없어&lt;br&gt;
절대 고백하지 마&lt;br&gt;
And it cut my heart&lt;br&gt;
open&lt;br&gt;
Put me&lt;br&gt;
in a coffin&lt;br&gt;
그 말과 동시에&lt;br&gt;
부서진 존심&lt;br&gt;
침착 gotta relax&lt;br&gt;
Hollywood 웃음 짓네&lt;br&gt;
Ha ha ha&lt;br&gt;
laugh like it&amp;rsquo;s funny&lt;br&gt;
오스카를 원해&lt;br&gt;
아카데미 원해&lt;br&gt;
긴장&lt;br&gt;
안 한 듯이&lt;br&gt;
영화배우처럼&lt;br&gt;
제스처는 큼직&lt;br&gt;
Ha ha ha&lt;br&gt;
laugh like it&amp;rsquo;s funny&lt;br&gt;
오스카를 원해&lt;br&gt;
아카데미 원해&lt;br&gt;
I&amp;rsquo;ll&lt;br&gt;
be the one&lt;br&gt;
니가 원하는 모든 역 다&lt;br&gt;
완벽하게 소화해 낼게&lt;br&gt;
I&amp;rsquo;ll&lt;br&gt;
be the one&lt;br&gt;
아무한테 안 보여 준&lt;br&gt;
내 모습 다&lt;br&gt;
너 원하는 건&lt;br&gt;
I&amp;rsquo;ll be the one&lt;br&gt;
Oh O O&lt;br&gt;
잠깐만이여&lt;br&gt;
Oh no O&lt;br&gt;
잠깐 wait me up&lt;br&gt;
어 아 음 길을 물으려&lt;br&gt;
하는 건 아냐&lt;br&gt;
사실&lt;br&gt;
할 말이 있어서&lt;br&gt;
오 오&lt;br&gt;
오늘 시간 있어요?&lt;br&gt;
Uh oh 없어도&lt;br&gt;
내게 와 줘요&lt;br&gt;
일이 많아도&lt;br&gt;
오 잠깐만 욜로 와 ay&lt;br&gt;
Okay 너를 데리러 갈게&lt;br&gt;
너는 어디야&lt;br&gt;
Oh baby&lt;br&gt;
그냥 call me up&lt;br&gt;
Olleh&lt;br&gt;
난 니 man이여&lt;br&gt;
엄마 다음으로 너를&lt;br&gt;
사랑해 girl 나한테 와&lt;br&gt;
수많은 생각들이&lt;br&gt;
내 머리를&lt;br&gt;
스칠 때마다 드는&lt;br&gt;
한 가지 비밀은&lt;br&gt;
나는 바로&lt;br&gt;
아마도 아 맞다&lt;br&gt;
사실 그게 음 I think&lt;br&gt;
I love you so much&lt;br&gt;
I&amp;rsquo;ll&lt;br&gt;
be the one&lt;br&gt;
니가 원하는 모든 역 다&lt;br&gt;
완벽하게 소화해 낼게&lt;br&gt;
I&amp;rsquo;ll&lt;br&gt;
be the one&lt;br&gt;
아무한테 안 보여 준&lt;br&gt;
내 모습 다&lt;br&gt;
너 원하는 건&lt;br&gt;
I&amp;rsquo;ll be the one&lt;br&gt;
I&amp;rsquo;ll&lt;br&gt;
be the one&lt;br&gt;
To give you&lt;br&gt;
the best of me&lt;br&gt;
Baby girl&lt;br&gt;
just leave it up to me&lt;br&gt;
Yeah you know that&lt;br&gt;
I&amp;rsquo;m the one to be yeah&lt;br&gt;
I&amp;rsquo;ll&lt;br&gt;
be the one&lt;br&gt;
To give you&lt;br&gt;
the best of me&lt;br&gt;
Baby girl&lt;br&gt;
just leave it up to me&lt;br&gt;
Yeah you know that&lt;br&gt;
I&amp;rsquo;m the one to be yeah&lt;br&gt;
네게 필요한&lt;br&gt;
역이 돼 줄게&lt;br&gt;
Whenever&lt;br&gt;
you need someone&lt;br&gt;
예상치 못할&lt;br&gt;
큰 감동을 줄게&lt;br&gt;
Whenever&lt;br&gt;
you need someone&lt;br&gt;
I&amp;rsquo;ll&lt;br&gt;
be the one&lt;br&gt;
니가 원하는 모든 역 다&lt;br&gt;
완벽하게 소화해 낼게&lt;br&gt;
I&amp;rsquo;ll&lt;br&gt;
be the one&lt;br&gt;
아무한테 안 보여 준&lt;br&gt;
내 모습 다&lt;br&gt;
너 원하는 건&lt;br&gt;
I&amp;rsquo;ll be the one&lt;/p&gt;

      </description>
      
      <category>pH-1</category>
      
      <category>Golden</category>
      
      <category>서동현(BIG Naughty)</category>
      
      <category>박재범</category>
      
    </item>
    
  </channel>
</rss>
